src/statsig/index.ts

```diff
@@ -52,3 +56,14 @@ export function getDynamicConfigParams<T extends Record<string, StatsigParameter
     return defaultValues
   }
 }
+
+export async function updateStatsigUser(statsigUser?: StatsigUser) {
```

Comment:
This makes an assumption that whatever is calling this with an updated property also updates the defaults in redux right? Should we document it?

Also, wonder if this could lead to the following race condition
1. Welcome screen updates startOnboardingTime and then updates redux store
2. Some user trait changes, which invokes identify
3. Identify invokes updateStatsigUser, but startOnboardingTime is not yet on redux store
4. StatsigUser no longer includes startOnboardingTime

Don't think this specifically can happen since the wallet address is created later in the flow, but this could be problem more generally if we have multiple custom properties and the redux store isn't updated with latest values right?

Reply:
If there's any chance at all of affecting the onboarding experiments, it's worth addressing right? If not, should be ok, but something to watch out for if/when we add more props

Reply:
i feel like this kind of problem is hard to fully avoid if we cache statsig user properties in redux with the goal of doing patch updates to the statsig user. without synchronously updating redux before calling updateStatsigUser, there's always some chance that the cache could be out of date when we update the statsig user. and I'm not really sure what to do about that.